### PR TITLE
SG-30477 App Session Launcher - Part 2: improve UI

### PR DESCRIPTION
**Pull request by julien-lang**
- Add feedback to widgets on hover, focus & clicked similarly to the SG web style
- Add confirmation dialog when closing the authentication window (close, cancel, escape)
- Remove cancel buttons

### CODE REVIEW

File: python/tank/authentication/resources/login_dialog.ui [hunk 1]
```diff
@@ -57,6 +58,21 @@ QComboBox
     margin-right: 3px;
 }
 
+QPushButton:focus
+{
+    border-color: rgb(48, 167, 227);
```
Comment (carlos-villavicencio-adsk) [inline python/tank/authentication/resources/login_dialog.ui:63]: It would be great if we can define color variables to improve UI maintainability.

**Review by carlos-villavicencio-adsk — APPROVED** [review]
Looks good. I just pointed out two small comments that are not blockers to moving forward.

File: python/tank/authentication/resources/login_dialog.ui (hunk 1 above)
Comment (julien-lang) [reply python/tank/authentication/resources/login_dialog.ui:63]: I believe QT5 only supports CSS2. Variables come in CSS3 :(

**Review by NorberMV — APPROVED** [review]
LGTM !

**Pull request merged**